packages/util/package.json

```diff
@@ -51,7 +51,7 @@
     "test:debug": "DEBUG=true yarn test"
   },
   "devDependencies": {
-    "@cardano-sdk/util-dev": "^0.4.1",
+    "@cardano-sdk/util-dev": "^0.5.0",
```

Comment:
@mkazlauskas is it expected we'll need to manually manage sibling deps? :thinking:

Reply:
No, this was probably an error with manual merge